### DISCUSSION TITLE
Prisma client not initialized (have run prisma generate)

### DISCUSSION THREAD

**Discussion by xorelia** [prisma/prisma#18808] [Q&A]
### Bug description

The code works fine on everyone elses computer, but fails on mine. Have version 4.12.0, and I run prisma generate before attempting to start the program. Yet it still fails? Anybody had a similar issue?

I am running everything on Ubuntu 20.04.5 LTS.

Thank you! 

### How to reproduce

Not sure

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: Ubuntu 20.04.5 LTS
- Database: PostgreSQL
- Node.js version: v18.15.0

### Prisma Version

```
prisma                  : 4.12.0
@prisma/client          : 4.12.0
Current platform        : debian-openssl-1.1.x
Query Engine (Node-API) : libquery-engine 659ef412370fa3b41cd7bf6e94587c1dfb7f67e7 (at node_modules/@prisma/engines/libquery_engine-debian-openssl-1.1.x.so.node)
Migration Engine        : migration-engine-cli 659ef412370fa3b41cd7bf6e94587c1dfb7f67e7 (at node_modules/@prisma/engines/migration-engine-debian-openssl-1.1.x)
Format Wasm             : @prisma/prisma-fmt-wasm 4.12.0-67.659ef412370fa3b41cd7bf6e94587c1dfb7f67e7
Default Engines Hash    : 659ef412370fa3b41cd7bf6e94587c1dfb7f67e7
Studio                  : 0.483.0
```

**Comment by janpio** [comment]
What do you mean with "it fails"?
What output does `prisma generate` give you?
Where should it generate to, and is it actually there after running `prisma generate` (default would be `node_modules/.prisma`)?

**Reply by nurul3101** [reply]
@xorelia Did you had a chance to look at Jan's response? Please let us know in case you need assistance.

**Reply by xorelia** [reply]
> What do you mean with "it fails"? What output does `prisma generate` give you? Where should it generate to, and is it actually there after running `prisma generate` (default would be `node_modules/.prisma`)?

Hi! Prisma generate works completely fine, giving the message 
```` 
✔ Generated Prisma Client (4.12.0 | library) to ./prisma/node_modules/@prisma/client in 290ms
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
```
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```
````

but then when i run it says:
```
server/node_modules/.prisma/client/index.js:3
    throw new Error(
    ^

Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
```

From what i can see the it is generated to the correct locations but still not working, thank you for answering!

**Reply by janpio** [reply]
The first `./prisma/` is unexpected in that success message from `prisma generate`.

Where are you triggering that command?
How is your project setup?
Are there maybe two `package.json` and/or `node_modules` in your proejct, once in the root and once in the `prisma` subfolder?

**Reply by nurul3101** [reply]
@xorelia Did you had a chance to look at Jan's last message? Let us know in case you need any assistance.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
